models/modelsmodule.h

```diff
@@ -49,6 +51,10 @@ class ModelsModule : public SLIModule
 
   const std::string name( void ) const;
   const std::string commandstring( void ) const;
+
+protected:
```

Comment:
Shouldn't it be private? There is no inheritance, all happens in the ModelsModule class.